src/components/VariationsAppBar/VariationsAppBar.js

```diff
@@ -12,48 +12,14 @@ import Close from '@material-ui/icons/Close';
 import Settings from '@material-ui/icons/Settings';
 import bem from '@fesk/bem-js';
 import './VariationsAppBar.scss';
-import posed, { PoseGroup } from 'react-pose';
 
 const $block = bem.block('variations-app-bar');
 
-const Anim = posed.span({
-  enter: { y: 0 },
-  exit: {
-    y: -40,
-  },
-});
-
-function SaveButton() {
-  const [isSaved, setIsSaved] = useState(false);
-
-  useEffect(
-    () => {
-      if (isSaved) {
-        setTimeout(() => {
-          setIsSaved(false);
-        }, 2000);
-      }
-    },
-    [isSaved]
-  );
-
-  return (
-    <PoseGroup>
-      {isSaved ? (
-        <Anim key="check">
-          <Check style={{ color: 'limegreen' }} />
-        </Anim>
-      ) : (
-        <Anim key="save">
-          <Save onClick={() => setIsSaved(true)} />
-        </Anim>
-      )}
-    </PoseGroup>
-  );
-}
-
 const VariationsAppBar = props => (
-  <div className={$block} position="static">
+  <div
+    className={props.noHeader ? $block + '-no-header' : $block}
```

Comment:
You can use the BEM helper to grab a modifier class name
```suggestion
    className={$block.modifiers({ 'no-header': props.noHeader })}
```

Note this will change the class name slightly from `variations-app-bar-no-header ` to `variations-app-bar--no-header` to match BEM naming

Reply:
We tried this change but it was setting both `variations-app-bar` and `variations-app-bar--no-header` as classes of the div instead of just `variations-app-bar--no-header`.  This led to styling issues on the buttons.

Reply:
unfortunately its a modifier, so both classes will be applied. Although you should be able to override any problematic styles